lib/scala/syntax/specialization/lexer-bench/src/bench/scala/org/enso/syntax/LexerBenchFixtures.scala

```diff
@@ -0,0 +1,205 @@
+package org.enso.syntax
+
+import java.nio.charset.StandardCharsets
+
+import org.enso.flexer
+import org.enso.flexer.Reader
+import org.enso.syntax.text.AST
+import org.enso.syntax.text.spec.{ParserDef, ParserDef2}
+
+object LexerBenchFixtures {
+
+  private val newEngine = flexer.Parser.compile(ParserDef())
+
+  // === Lexer Runner ===
+
+  def runLexer(input: String): ParserDef2.Result[AST.Module] = {
+    val engine = newEngine()
+//    val reader = new Reader(new CommentRemover(input).run)
+    val reader = new Reader(input)
+    engine.run(reader)
+  }
+
+  // === Utilities ===
+
+  def replicate(
+    input: String,
+    size: Int,
+    addNewline: Boolean = false
```

Comment:
A documentation comment may be helpful here and for other helper methods (`preprocess`, `runLexer`) to better understand their behaviour.